R/long_est.R

```diff
@@ -0,0 +1,847 @@
+long_est_1 <- function(long_data, model, LM_fixedEffect_withTime_variables, LM_randomEffect_variables, optCtrl = list(method = "nlminb", kkt = FALSE, tol = 0.2, maxit = 20000)) {
```

Comment:
Depending on the user input of `method`, we show them a warning message which inputs they need to provide for each method.

```suggestion
long_est_1 <- function(long_data, method, LM_fixedEffect_withTime_variables, LM_randomEffect_variables, optCtrl = list(method = "nlminb", kkt = FALSE, tol = 0.2, maxit = 20000)) {
```